src/itsdangerous/serializer.py

```diff
@@ -1,12 +1,26 @@
 import json
+import typing as _t
 
 from .encoding import want_bytes
 from .exc import BadPayload
 from .exc import BadSignature
+from .signer import _make_keys_list
 from .signer import Signer
 
+if _t.TYPE_CHECKING:
+    _t_str_bytes = _t.Union[str, bytes]
```

Comment:
Would it be better to stick commonly used types in a separate module/file?

Reply:
I thought about this, but ultimately decided it wasn't worth the further indirection.

> Only a few types (really just `str_bytes`) were common between modules, didn't bother with a common `_typing` module.

For a bigger library it might make more sense.

Reply:
That makes sense. I find that it helps to stick them in the same file and for all usages to reference a single definition because it encourages consistency.